packages/components/e2e/component-objects/src/main/java/org/talend/component/Breadcrumb.java

```diff
@@ -0,0 +1,88 @@
+package org.talend.component;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.openqa.selenium.By;
+import org.openqa.selenium.NotFoundException;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+
+import java.util.Iterator;
+import java.util.List;
+
+public class Breadcrumb extends Component {
+
+    private static final Logger LOGGER = LogManager.getLogger(Breadcrumb.class);
+
+    static final String NAME = "Breadcrumb";
+
+    static final String SELECTOR = ".tc-breadcrumb";
+
+    static final String ITEM_SELECTOR = ".tc-breadcrumb-item";
+
+    static final String ACTIVE_ITEM_SELECTOR = ITEM_SELECTOR + ".active";
+
+    static final String COLLAPSED_MENU_BUTTON_SELECTOR = ".dropdown-toggle";
+
+    static final String COLLAPSED_MENU_SELECTOR = ".dropdown-menu";
+
+    static final String COLLAPSED_MENU_ITEM_SELECTOR = COLLAPSED_MENU_SELECTOR + " a";
+
+    /**
+     * Breadcrumb constructor
+     *
+     * @param driver Selenium WebDriver
+     */
+    Breadcrumb(WebDriver driver) {
+        super(driver, NAME, SELECTOR);
+    }
+
+    public List<WebElement> getItems() {
+        LOGGER.info(NAME + ".getItems");
+        return this.getElement().findElements(By.cssSelector(ITEM_SELECTOR));
+    }
+
+    public WebElement getItem(String label) {
```

Comment:
IMO we should expose a `public void clickOnItem(final String label)`. In practice QA wants to get the item to click on it.